challenge-04/README.md

```diff
@@ -7,15 +7,36 @@ um único parâmetro como argumento. Essa função deve retornar `true` se o
 equivalente booleano para o valor passado no argumento for `true`, ou `false`
 para o contrário.
 */
-?
+var isTruthy = function(parm) {
+  if (parm) {
+    return true;
+  } else {
```

Comment:
-1
Quando você usa um `return` no `if`, não é necessário usar `else`, pois a avaliação desse código pára por ali ;)
E se o valor no `if` for false, o interpretador do JS já pula para a próxima instrução :)